pandas/core/frame.py

```diff
@@ -5022,10 +5029,8 @@ def pivot(self, index=None, columns=None, values=None):
         1  one   A    2
         2  two   B    3
         3  two   C    4
-
         Notice that the first two rows are the same for our `index`
         and `columns` arguments.
-
```

Comment:
Can you also undo this removal of blank lines?

Reply:
All fixed now @jorisvandenbossche.